invenio_rdm_records/config.py

```diff
@@ -378,6 +387,29 @@ def always_valid(identifier):
     }
 """
 
+# Configuration for the OSTIClient used by the OSTIPIDProvider
+OSTI_ENABLED = False
+"""Flag to enable/disable DOI registration to OSTI."""
+
+
+OSTI_USERNAME = ""
+"""OSTI username."""
+
+
+OSTI_PASSWORD = ""
+"""OSTI password."""
+
+OSTI_ACCESSION_NUMBER_PREFIX = ""
+"""Prefix to use to generate OSTI accession_num to uniquely identify records that have gotten DOI's reserved"""
+
+OSTI_TEST_MODE = True
+"""OSTI test mode enabled."""
+
+# until changes can be made in the the UI require RDM instances to set in their invenio.cfg these 2 OSTI required metadata fields
```

Comment:
What kind of changes would be needed? change the DOI widget to allow changing those?

Reply:
That comment was for the below OSTI_CONTRACT_NOS and OSTI_SPONSER_ORG configs.  For our instance of RDM it was fine for all records to share the same value for those 2 fields when minting DOI's but I'm not sure if that would always be the case.  If not, custom metadata fields would need to be added to the deposit form to collect them from the user.